src/components/AppMap.ts

```diff
@@ -834,9 +836,50 @@ export default class AppMap extends mixins(MixinUtil) {
       marker.data.getMarker().addTo(this.map.m);
     }
 
+    this.updateTooltip();
     this.searching = false;
   }
 
+  yTooltipOnEach(marker: any) {
+    let m: any = marker.getMarker();
+    if (!('_tooltip' in marker.obj)) {
+      // @ts-ignore
+      let tt = m.getTooltip();
+      marker.obj._tooltip = tt.getContent();
+      marker.obj._tooltip_options = tt.options;
+    }
+    m.unbindTooltip();
+    m.bindTooltip(`${marker.obj.pos[1]}`, { permanent: true });
+    m.openTooltip();
+  }
+
+  yTooltipOffEach(marker: any) {
+    let m: any = marker.getMarker();
+    m.getTooltip().options.permanent = false;
+    m.unbindTooltip();
+    m.bindTooltip(marker.obj._tooltip, marker.obj._tooltip_options);
+    m.closeTooltip();
+  }
+
+  yTooltipSwitch(on: boolean) {
```

Comment:
toggleYTooltipOnAllMarkers (or something else that makes it clear that this function operates on every single map marker)